mne/io/brainvision/brainvision.py

```diff
@@ -836,3 +836,16 @@ def read_raw_brainvision(vhdr_fname, montage=None,
     return RawBrainVision(vhdr_fname=vhdr_fname, montage=montage, eog=eog,
                           misc=misc, scale=scale, preload=preload,
                           stim_channel=stim_channel, verbose=verbose)
+
+
+def _bv_parser(description, add_for_r=1000):
+    """Parse BrainVision event codes (like `Stimulus/S 11`) to ints."""
+    try:
+        event_type, code = description[:-3], int(description[-3:])
+        if event_type in ("Response/R", "Stimulus/S"):
+            if event_type == "Response/R":
+                code += add_for_r
+            return code
+    except ValueError:  # probably not coercable to int
```

Comment:
just for the record, this function has no return value. We should try to avoid multiple return points. (or make sure that all branches do return)